data/common.yaml

```diff
@@ -0,0 +1,88 @@
+simp::scenario_map:
+  none: []
+  poss:
+    - pupmod
+  simp_lite:
+# Shared with poss
+    - pupmod
+# Shared with simp
+    - simp::scenario::base
+    - aide
+    - auditd
+    - clamav
+    - chkrootkit
+    - at
+    - cron
+    - incron
+    - useradd
+    - resolv
+    - nsswitch
+    - issue
+    - tuned
+    - swap
+    - timezone
+    - ntpd
+    - pam::access
+    - simp_rsyslog
+    - simp::admin
+    - simp::base_apps
+    - simp::base_services
+    - simp::yum
+    - simp::kmod_blacklist
+    - simp::mountpoints
+    - simp::sysctl
+    - ssh
+    - sudosh
+  simp:
+# Shared with poss
+    - pupmod
+# Shared with simp_lite
+    - simp::scenario::base
+    - aide
+    - auditd
+    - clamav
```

Comment:
I'm told this is an accident and should be in simp::scenario::base and gated by booleans